cypress/plugins/index.js

```diff
@@ -10,4 +10,18 @@
 module.exports = ( _on, _config ) => {
```

Comment:
Well, now both `_on` and `_config` were changed to remove the underscore, but `config` is unused (as pointed out by eslint), so that probably should’ve stayed as `_config`.

Reply:
You are right, @lucaswerkmeister . The new commit has `on` and `_config`